contributed/lora_bwd.py

```diff
@@ -0,0 +1,281 @@
+import sys
+
+from neuronxcc import nki
+import neuronxcc.nki.language as nl
+import neuronxcc.nki.isa as ni
+import numpy as np
+
+def lora_bwd(I_DRAM, PW_DRAM, A_DRAM, B_DRAM, AI_DRAM, dO_DRAM, alpha):
```

Comment:
Can we get documentation on expected shape and important highlights of the algorithm approach etc...?